Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1328,6 +1328,51 @@ def stringUnEscape(st):
     return st.replace('\\r', '\r').replace('\\n', '\n').replace('\\t', '\t')
 
 
+def return_next_interval(fetch_interval: str, last_update_time: datetime):
+    """
+        Returns a datetime object of the next update time. For example, when the function is given a last update
+        datetime, this function will return the next datetime object for the given interval.
+
+        :type fetch_interval: ``str``
+        :param fetch_interval: A string containing a date range e.g. (2 Days)
+
+        :type last_update_time: ``datetime``
+        :param last_update_time: The datetime object that the date range will be incremented on.
+
+        :return: A datetime object for the next interval.
+        :rtype: ``datetime``
+    """
+    range_split = " ".join(fetch_interval.split()).strip().split(' ')
```

Comment:
Would this work? One strip and one split (instead of join, strip, and 2 splits)
```
fetch_interval_split = fetch_interval.strip().split()
number = fetch_interval_split[0]
unit = fetch_interval_split[-1].lower()
```